project/src/types/live-place.ts

```diff
@@ -0,0 +1,11 @@
+export type LivePlaceProps = {
+  id: string,
+  isPremium: boolean,
+  isBookmarked: boolean,
+  placeType: string,
+  price: number,
+  rating: number,
+  name: string,
+  photoSrc: string,
+  city: string,
+}
```

Comment:
```suggestion
type LivePlace = {
  id: string,
  isPremium: boolean,
  isBookmarked: boolean,
  placeType: string,
  price: number,
  rating: number,
  name: string,
  photoSrc: string,
  city: string,
}

export type { LivePlace };
```
Это не пропсы, это уже как отдельная сущность. Пропсы ассоциируются с пропсами в реакт компонентов.

Reply:
Экспорты только внизу

Reply:
Не забудь енам PlaceType для типа использовать